src/StaticFAQ/FAQArticleDetailContent.js

```diff
@@ -5,58 +5,65 @@ import Markdown from 'markdown-to-jsx';
 import { Typography } from '@kiwicom/orbit-components';
 import { createFragmentContainer, graphql } from 'react-relay';
 
-import type { FAQArticleDetailContent_article } from './__generated__/FAQArticleDetailContent_article.graphql';
+import Breadcrumbs from './Breadcrumbs';
+import type { FAQArticleDetailQuery } from './__generated__/FAQArticleDetailQuery.graphql';
 
-type Props = {
-  article: FAQArticleDetailContent_article,
-};
+type Props = FAQArticleDetailQuery;
 
-const Detail = ({ article }: Props) => (
-  <div className="faq-article-content">
-    <Typography size="header" type="attention">
-      {article.title}
-    </Typography>
-    <div className="faq-article-perex">
-      <Typography type="attention" variant="bold">
-        Summary:
+const Detail = ({ article, category }: Props) => (
+  <React.Fragment>
+    <div className="faq-article-content">
+      <div>
+        <Breadcrumbs
```

Comment:
Breadcrumbs don't need to be in this component, moving them to `renderDetailContent` might help you to define relay fragments in a better way